OvieMudi/Banka
# #164979262 client dashboard page

#### What does this PR do?
Creates a dashboard UI page where a user (client) can
 - view all accounts info
 - navigate to a page where the user can create a new account
#### How should this be manually tested?
- Navigate to the page at https://oviemudi.github.io/Banka/UI/dashboard-client
- Ensure page is responsive and meets UI requirements
#### Any background context you want to provide?
A  user will get easy access to operations on the app from this **dashboard page**
#### What are the relevant pivotal tracker stories?
[#164979262](https://www.pivotaltracker.com/story/show/164979262)
